### PR TITLE
Linear design system updates

### PR DESCRIPTION
**Pull request by dpiatek**
Jira Ticket Link / Motivation
----

Implement:
https://app.asana.com/0/1202054865216493/1202146409149931/f
https://app.asana.com/0/1202054865216493/1202146409149933/f

Additionally:
- update paragraph text styles to correct color
- move Source Code Pro import to a separate file

Summary of changes
----

See commits for details.

How do you manually test this?
----

All changes can be review in the preview app (remotely or locally)

Reviewer Tasks (optional)
----

Merge/Deploy Checklist
----

- [ ] Written automated tests for implemented features/fixed bugs
- [ ] Rebased and squashed commits
- [ ] Commits have clear descriptions of their changes
- [ ] Checked for any performance regressions

Frontend Checklist
----

- [ ] No frontend changes in this PR
- [ ] Added before/after screenshots for changes
- [ ] Tested on different platforms/browsers with [Browserstack](https://www.browserstack.com)
- [ ] Compared with the initial design / our [brand guidelines](https://www.figma.com/file/jTQgyIovrhGBOf4Zrvjvbk/Ably-Linear-Design?node-id=118%3A0)
- [ ] Checked the code for accessibility issues ([VoiceOver User Guide](https://support.apple.com/en-gb/guide/voiceover/welcome/mac))?

### CODE REVIEW

File: src/core/styles/properties.css [hunk 1]
```diff
@@ -15,9 +15,10 @@
     --color-charcoal-grey: #292831;
     --color-gui-default: #0073e6;
     --color-gui-hover: #0867c4;
-    --color-gui-focus: #80b9f2;
+    --color-gui-focus: #0073e6;
+    --color-gui-focus-outline: #80b9f2;
     --color-gui-active: #074095;
-    --color-gui-viewed: #4887c2;
+    --color-gui-visited: #4887c2;
```
Comment (artismarti) [inline src/core/styles/properties.css:21]: This needs to be applied.

File: src/core/styles/text.css [hunk 2]
```diff
@@ -146,21 +146,17 @@
...
-    @apply text-gui-viewed;
-  }
-
   .ui-link {
-    @apply hover:text-active-orange active:text-red-orange;
-    text-decoration-color: var(--color-active-orange);
-    text-underline-offset: 4px; /* px used here as behaves weird with rem's */
-    text-decoration-line: underline;
-    text-decoration-thickness: 0.125rem;
+    @apply visited:text-gui-viewed;
```
Comment (artismarti) [inline src/core/styles/text.css:150]: this needs to be `visited` :)

**Review by artismarti — CHANGES_REQUESTED** [review]
as discussed on Tuple! :)

**Review by artismarti — APPROVED** [review]
🚀 
LGTM

**Review by aralovelace — APPROVED** [review]

**Pull request merged**